src/sst/elements/balar/tests/vanadisLLVMRISCV/balar_vanadis.h

```diff
@@ -0,0 +1,47 @@
+// Copyright 2009-2023 NTESS. Under the terms
+// of Contract DE-NA0003525 with NTESS, the U.S.
+// Government retains certain rights in this software.
+//
+// Copyright (c) 2009-2023, NTESS
+// All rights reserved.
+//
+// Portions are copyright of other developers:
+// See the file CONTRIBUTORS.TXT in the top level directory
+// of the distribution for more information.
+//
+// This file is part of the SST software package. For license
+// information, see the LICENSE file in the top level directory of the
+// distribution.
+
+/**
+ * @file cuda_runtime_api.h
+ * @author Weili An (an107@purdue.edu)
```

Comment:
@William-An You should probably remove your email address from these unless you want users bugging you directly. ^-^